src/webgpu/shader/validation/shader_io/align.spec.ts

```diff
@@ -13,7 +13,10 @@ const kTests = {
   },
   one: {
     src: '@align(1)',
-    pass: true,
+    pass: false,
+    // EXCEPTION: Error: Unexpected validation error occurred:
+    // Error while parsing WGSL: :6:10 error: alignment must be a
+    // multiple of '4' bytes for the 'uniform' address space @align(1) a: i32,
```

Comment:
yes, this LGTM.

Reply:
whoops, no this is wrong.

The real check is whether the offset if incorrect.

Reply:
Based on our discussion it sounds like this change is correct.
Shall i remove the extraneous comment in the source

Reply:
ok, after a bunch of discussion, we agreed to change Tint to be more in line with Naga and WebKit.

crbug.com/379760170  is the Tint change.

But also clarify the spec: https://github.com/gpuweb/gpuweb/pull/4978

Reply:
LGTM.  

Given the proposed spec change, additional tests will be needed.  For followup.